Tests/SwiftTryCatchTests/SwiftTryCatchTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SwiftTryCatch.h
+//
+//  Created by Dan Federman on 12/20/19.
+//  Copyright (c) 2019 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftTryCatch
+import XCTest
+
+final class SwiftTryCatchTests: XCTestCase {
+
+    func test_try_catchesObjectiveCFailure() {
+        let doomedFileHandle = FileHandle(fileDescriptor: 101)
+        var didCatch = false
+        SwiftTryCatch.try({
+            // This will raise an exception.
+            doomedFileHandle.readData(ofLength: 5)
+        }, catch: { exception in
+            XCTAssertNotNil(exception)
+            didCatch = true
+        })
+        XCTAssertTrue(didCatch)
+    }
+
+    func test_try_stopsExecutingOnRaise() {
+        let doomedFileHandle = FileHandle(fileDescriptor: 101)
+        var didExecuteAfterRaisedException = false
+        SwiftTryCatch.try({
+            // This will raise an exception.
+            doomedFileHandle.readData(ofLength: 5)
+            didExecuteAfterRaisedException = false
+        }, catch: { _ in })
+        XCTAssertFalse(didExecuteAfterRaisedException)
```

Comment:
This tests is not ideal since we set `didExecuteAfterRaisedException` as false initally, and then we go to a closure, so in theory if this closure were to become async we would not get in the closure and this test will still succeed.

Also, we're setting `didExecuteAfterRaisedException = false` (even though it starts as false) in line 44, so we're actually not modifying anything and then checking if the original value is still set.

I'd suggest something like this instead:

```swift
let doomedFileHandle = FileHandle(fileDescriptor: 101)
var didStopAfterRaisedException = false
SwiftTryCatch.try({
    didStopAfterRaisedException = true
    // This will raise an exception.
    doomedFileHandle.readData(ofLength: 5)
    didStopAfterRaisedException = false
}, catch: { _ in })

XCTAssertTrue(didStopAfterRaisedException)
```